inst/ms/aggreCAT.qmd

```diff
@@ -1355,7 +1476,7 @@ IDEA protocol has been used to elicit judgements.
 
```

Comment:
L1473: Haven't tested this, but could they list as `round_1` and toggle `round_two_filter` to `FALSE` and still get them to work?

Reply:
`round_two_filter` currently not included as argument to aggregation functions. Has this feature been removed or do I need to rebase onto master? 
But does seem like a good idea, seems a bit restrictive to force the user to write 'round_2' and also is erroneous.

Reply:
- [ ] re-expose `round_two_filter` arg in wrapper functions

Reply:
Can confirm that if only `round_1` Vals supplied, and `round_two_filter` set to `FALSE` that CS can still be calculated:

``` r
library(aggreCAT)
#> ══ aggreCAT ════════════════════════════════════════════════════════════════════
#> Version: 0.0.0.9002 
#> Please do not feed the cat. 
#> ════════════════════════════════════════════════════════════════════════════════
library(tidyverse)

data_ratings %>%
  filter(round != "round_2") %>%
  AverageWAgg(round_2_filter = FALSE)
#> 
#> ── AverageWAgg: ArMean ─────────────────────────────────────────────────────────
#> 
#> ── Pre-Processing Options ──
#> 
#> ℹ Round Filter: FALSE
#> ℹ Three Point Filter: TRUE
#> ℹ Percent Toggle: FALSE
#> # A tibble: 25 × 4
#>    method paper_id    cs n_experts
#>    <chr>  <chr>    <dbl>     <int>
#>  1 ArMean 100       63.4        25
#>  2 ArMean 102       38.4        25
#>  3 ArMean 103       65.5        25
#>  4 ArMean 104       55.8        25
#>  5 ArMean 106       40.3        25
#>  6 ArMean 108       70.1        25
#>  7 ArMean 109       73          25
#>  8 ArMean 116       63.4        25
#>  9 ArMean 118       51.6        25
#> 10 ArMean 133       65.1        25
#> # … with 15 more rows
```

<sup>Created on 2023-02-14 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
Fixed in #37